README.md

```diff
@@ -1,54 +1,45 @@
 # PromClient.jl
 Lightweight Julia Implementation of a Prometheus Client for Julia services. 
 
-At the moment it only supports:
+At the moment it supports:
 - Gauge
 - Counter
-
-# TODOs before we publish initial version
 - Histogram
-- Stop users from breaking labels
-- Default prom collector
-- optional timestamps after metrics
-
-Not yet supported Metric types:
+## Not yet supported Metric types:
 - Summary 
 
+## To Do's in future (not yet supported)
+- Default System Metrics (CPU time, GC time, memory use etc.).
+- Decorators for code timers directly into metrics.
+
 For further on metric types see https://prometheus.io/docs/concepts/metric_types/
 
-# Quickstart
-git remote add upstream https://github.com/spcogg/PromClient.jl.git
-See Tests for more detailed examples
+# Quick Start
+See Tests for more detailed examples.
 
 ```
-# create a new metric. Defaults to int, use the vtype kwarg for other numeric types
+# Create a new metric. Defaults to int, use the vtype kwarg for other numeric types
 julia> metric1 = CounterMetric("test:Counter", "Helpful Description String", ["hostname", "port", "threadid"])
 
-julia> add_metric_to_collector!(metric1)  # adds to default collector, used to generate the metric output
+julia> add_metric_to_collector!(metric1)  # Adds to default collector, used to generate the metric output
```

Comment:
Does this still need to be called? I think we put this call into the constructor right

Reply:
So either remove this, or change it to an example for using a custom collector

Reply:
I'll remove it, but I noticed in the unit-tests we are adding it to default collector again as seen on the (screenshot below), looks like legacy code and might need to refactor our unit-tests.
![image](https://user-images.githubusercontent.com/26830876/197694689-76f20d36-28cf-487a-b598-02205a0923cd.png)

Reply:
the test won't do anything right now since it already gets added to the default collector in the constructor.
Yes you can remove it - might be a general tersting/cleanup job we do.